lib/foundation_rails_helper/flash_helper.rb

```diff
@@ -18,29 +18,29 @@ module FlashHelper
       :warning   => :warning,
       :primary   => :primary
     }
-    def display_flash_messages(key_matching = {})
-      key_matching = DEFAULT_KEY_MATCHING.merge(key_matching)
+    def display_flash_messages(**options)
```

Comment:
I originally wanted to set `closable` as a keyword argument just as @dgmstuart suggests above, but as you mention it would be a breaking change. How many other breaking changes are there in the Foundation 6 branch?